island-is/island.is
# feat(samradsgatt): Case details page progress timeline

# Add progress timeline to case details page

## What

Add progress timeline to case details page

## Why

It is needed

## Screenshots / Gifs

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/106164521/213650933-397957a9-06e0-487a-8a2f-11489dd651d9.png">

https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=68%3A4030&t=aSXASoH0qK1xFHGZ-0

## Leftovers

Still missing: The sections that are completed are faded out and have the numbers one them not check mark.

## How to test

- Checkout branch feature/samradsgatt/KAM-593-details-page-case-timeline-left-side-column
- Run the project locally : yarn start samradsgatt
- Navigate to http://localhost:4200/samradsgatt/Cases/id
- Look at the top left side case timeline
- Should look similar to the one in the figma design

## Checklist:
